internal/prompter/prompter.go

```diff
@@ -97,6 +99,22 @@ func (p *surveyPrompter) Input(prompt, defaultValue string) (result string, err
 	return
 }
 
+func (p *surveyPrompter) ConfirmDeletion(requiredValue string) error {
+	var result string
+	return p.ask(
+		&survey.Input{
+			Message: fmt.Sprintf("Type %s to confirm deletion:", requiredValue),
+		},
+		&result,
+		survey.WithValidator(
+			func(val interface{}) error {
+				if str := val.(string); !strings.EqualFold(str, requiredValue) {
```

Comment:
Nice that we're abstracting a convoluted Survey call behind a simple interface that takes just a string and returns an error 🎉